### PR TITLE
Shanice contactform

### PR DESCRIPTION
**Pull request by shanicenaomihaye**

### CODE REVIEW

File: frontend/src/ContactForm/contact.css
Comment (mehtapparkinson) [inline frontend/src/ContactForm/contact.css:1]: When i am on your branch I can see the css that you build for the form however it seems like it was not committed here. Could you please take a look @shanicenaomihaye ?

Comment (shanicenaomihaye) [reply frontend/src/ContactForm/contact.css:1]: I made 2 separate commits, the css is in the second commit @mehtapparkinson

**Review by mehtapparkinson — CHANGES_REQUESTED** [review]
Hi Shanice wonderful job, very well-structured. could we please adjust the current code with React Bootstrap component names for the unity of the project and it would be helpful for mobile view. Thank you
[https://react-bootstrap.netlify.app/docs/forms/overview]

File: frontend/src/ContactForm/contact.css
Comment (mehtapparkinson) [reply frontend/src/ContactForm/contact.css:1]: Oh perfect just seen now thanks 🙏

File: frontend/src/ContactForm/ContactForm.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import './ContactForm.css';
```
Comment (mehtapparkinson) [inline frontend/src/ContactForm/ContactForm.jsx:2]: css file name does not seem to match @shanicenaomihaye

**Pull request closed without merging**